Code/Editor/EditorFramework/Assets/AssetWatcher.h

```diff
@@ -51,7 +51,7 @@ class EZ_EDITORFRAMEWORK_DLL ezAssetWatcher
   ezHybridArray<ezDirectoryWatcher*, 6> m_Watchers;
   ezSharedPtr<ezTask> m_pWatcherTask;
   ezTaskGroupID m_WatcherGroup;
-  ezAtomicBool m_bShutdown = false;
+  ezAtomicBool m_Shutdown = false;
```

Comment:
Should we do the same prefixes for atomics as for regular vars? I have no preference tbh.